rootfs/api/models/app.py

```diff
@@ -211,6 +211,17 @@ def create(self, *args, **kwargs):  # noqa
             except KubeException:
                 self._scheduler.ns.create(namespace)
 
+            if settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC != '':
+                quota_name = '{}-quota'.format(namespace)
+                self.log(settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC)
+                quota_spec = json.loads(settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC)
+                self.log('creating Quota {} for namespace {}'.format(quota_name, namespace),
+                         level=logging.DEBUG)
+                try:
+                    self._scheduler.quota.get(namespace, quota_name)
```

Comment:
The use case is the following:
- Admin sets default quota
- Users create applications (default quota is applied for each application)
- One of the users need more resources for particular application (my-shiny-app)
- Admin changes quota for my-shiny-app namespace with help of kubectl
With hundreds of users and applications it is impossible to run production installation without default quota - it is guaranteed someone will ruin the cluster by using all available memory and CPU.